CLAUDE.md

```
@@ -178,6 +234,175 @@ async def launch_new_workflow(self, worktree_path: Path, issue_number: int) -> W
+
+### How Integration Branch Works
+
+The build command creates a temporary integration branch that merges all stack components:
+
+1. **Creates branch**: `epic-580-build` from `main`
+2. **Merges stacks**: Merges all stack top branches (e.g., `issue-585`)
+3. **Handles parallel stacks**: If epic has multiple independent stacks, merges all of them
+4. **Tests integration**: Runs `build-dev.sh` to create container with complete epic
+
+**Branch structure example**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)